docs/ref/templates/builtins.txt

```diff
@@ -1877,18 +1877,19 @@ For example, you can apply ``escape`` to fields when :ttag:`autoescape` is off:
 ``escapejs``
 ------------
 
-Escapes characters for use in JavaScript strings. This does *not* make the
-string safe for use in HTML or JavaScript template literals, but does protect
-you from syntax errors when using templates to generate JavaScript/JSON.
+Escapes characters for use as a whole JavaScript string literal, within single
+or double quotes, as below. This filter does not make the string safe for use
+in *"JavaScript template literals"* (the JavaScript backtick syntax), any other
+use is not supported. It is generally recommended that data should be passed
```

Comment:
> This filter does not make [...], any other use is not supported

I feel like there's something missing before the `any other use` - as it stands, that sounds like the previous item(s) were a list of _supported_ use cases. 

i.e. it's actually: "this doesn't make it safe for template literals, **_nor_** any other usages not enumerated above"

That isn't a suggested wording, but I can't find a non-awkward way to say it right now 😄

Reply:
Maybe as a separate sentence:

_"This filter does not make the string safe ... . Any other uses not listed above are not supported. It is generally recommended ..."_

Reply:
@kezabelle I pushed edits, please take a look.

Reply:
It's still a _little_ awkward, because the ordering still _kinda_ suggests that the previous item(s) were all supported use cases/behaviours. 
We technically _could_ probably solve that by moving the `Any other uses` to be _before_ the admonition about backtick syntax, but I think it's actually more important to front-load that part as "hey this isn't supported" ... so on balance I think it's fine as-is and anything more is just my tilting at windmills.

Reply:
> It's still a _little_ awkward, because the ordering still _kinda_ suggests that the previous item(s) were all supported use cases/behaviours. We technically _could_ probably solve that by moving the `Any other uses` to be _before_ the admonition about backtick syntax, but I think it's actually more important to front-load that part as "hey this isn't supported" ... so on balance I think it's fine as-is and anything more is just my tilting at windmills.

Yeah, it's really hard to find the right balance in wording (check out #15430). Let's move it forward, thanks :+1: